src/gateways/mappers/BeaconResponseMapper.ts

```diff
@@ -17,11 +18,17 @@ export class BeaconResponseMapper implements IBeaconResponseMapper {
       manufacturer: beaconApiResponse.data.attributes.manufacturer,
       model: beaconApiResponse.data.attributes.model,
       status: beaconApiResponse.data.attributes.status,
-      registeredDate: beaconApiResponse.data.attributes.createdDate,
-      batteryExpiryDate: beaconApiResponse.data.attributes.batteryExpiryDate,
+      registeredDate: yyyyMmDdFormat(
+        beaconApiResponse.data.attributes.createdDate
+      ),
+      batteryExpiryDate: yyyyMmDdFormat(
+        beaconApiResponse.data.attributes.batteryExpiryDate
+      ),
       chkCode: beaconApiResponse.data.attributes.chkCode,
       protocolCode: beaconApiResponse.data.attributes.protocolCode,
-      lastServicedDate: beaconApiResponse.data.attributes.lastServicedDate,
+      lastServicedDate: yyyyMmDdFormat(
+        beaconApiResponse.data.attributes.lastServicedDate
+      ),
```

Comment:
As commented above on the `singleBeaconFixture`: time information is stripped out from the date fields at the boundary.